tests/monitors/custom_collectd/custom_collectd_test.py

```diff
@@ -78,3 +88,30 @@ def test_custom_collectd_shutdown():
         """))
 
         assert wait_for(p(has_datapoint_with_dim, backend, "plugin", "ping")), "Didn't get ping datapoints"
+
+
+@pytest.mark.k8s
+@pytest.mark.kubernetes
+def test_custom_collectd_in_k8s(agent_image, minikube, k8s_test_timeout, k8s_namespace):
```

Comment:
Remove, there is no value testing this monitor in k8s since it is not directly a service.